iOS/rainbow/Controller/Camera/CameraController.swift

```diff
@@ -37,7 +37,7 @@ class CameraController: LuminaViewController {
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
         self.delegate = self
-        self.streamingModelTypes = [ProjectRainbowModel_1753554316()]
+        self.streamingModels = [LuminaModel(model: ProjectRainbowModel_1753554316().model, type: "WatsonML")]
```

Comment:
Just FYI, we are still using a branch of Lumina, and not yet the master copy. I will be, at some point, evaluating these changes for a merge into master.